webpack.common.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    libraryTarget: 'umd',
+    library: 'LinxBanner',
```

Comment:
Check if it is possible to export to `window.linx.banner`. If it is not possible, I think that it is better to do it manually on the `index.js`.

Something like:

```javascript
import { module };

const LinxBanner = {
  module,
};

window.linx = window.linx || {}:
window.linx.banner = LinxBanner;
```